package.json

```diff
@@ -71,11 +71,13 @@
     "ansi-html": "mahdyar/ansi-html-community#v0.0.8",
     "ansi-regex": "^5.0.1",
     "async": "^2.6.4",
+    "ember-source-channel-url": "^3.0.0",
```

Comment:
should this be in resolutions since it's in dev above?

Reply:
Yes. Since other packages also depend on it, we have to resolve the version even though it's a top-level dependency for us.